style.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  position: relative;
+  background-color: #fff;
+}
+
+header {
+  color: #6070ff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1em 7%;
+}
+
+.logo > a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.headline-main {
+  max-height: 800px;
+  min-height: 100vh;
+  width: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0 7%;
+  background-image: url(./assets/header-shape.png);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.headline {
+  margin: 10px 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  height: 100%;
+}
+
+.headline-heading {
+  margin: 10px 0;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 52px;
+  display: flex;
+}
+
+.headline-paragraph {
+  margin: 10px 0;
+  color: #172b4d;
```

Comment:
- [ ] Based on the template CSS given, use the color `#344563` for the paragraph here.

## View given:
![image](https://user-images.githubusercontent.com/50990925/219018986-3f3ef630-6dc9-4901-8126-8b7413563ef2.png)

Reply:
I updated it as stated.